controls/gridview/rows/summary-rows.md

```diff
@@ -237,67 +240,66 @@ End Sub
 
 ![gridview-rows-summary-rows 006](images/gridview-rows-summary-rows006.png)
 
-The same event is used to format the group header so it is important to check the __Parent__ in the event arguments, as shown in the above example.
         
-## Extending the GridViewSummaryItem
```

Comment:
It looks like this heading refers to the same thins as the _## SummaryItem properties_ heading above. We must use the same name when we refer to the same thing in a document.